Library/Homebrew/formula.rb

```diff
@@ -1548,6 +1548,15 @@ def link_overwrite?(path)
   # @see .disable!
   delegate disable_reason: :"self.class"
 
+  # Sandbox rules that should be skipped when installing or testing this {Formula}.
+  # Returns `nil` if there are no sandbox rules to skip.
+  # @!method reduced_sandbox
+  # @return [Array<Symbol>]
+  # @see .reduce_sandbox
+  def reduced_sandbox
```

Comment:
```suggestion
  def sandbox_allow
```

or something is more obvious naming to me than `reduced`